### PR TITLE
Npm free asset without npm

### PR DESCRIPTION
**Pull request by f1amy**
Using npm-asset without need in npm.
@rmrevin, can you look at my code and see if this can be implemented?

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -24,7 +24,8 @@
   ],
   "require": {
     "php": ">=5.4.0",
-    "yiisoft/yii2": "^2.0.0"
+    "yiisoft/yii2": "^2.0.0",
+    "npm-asset/fortawesome--fontawesome-free": "~5.7.1"
```
Comment (rmrevin) [inline composer.json]: Not sure if unconditional installation of the package is appropriate. Many people use the pro version, they do not need free.

Comment (f1amy) [reply composer.json]: Ok I'll remove it and update docs

**Comment by f1amy** [comment]
I updated PR, removed unconditional installation and updated docs.
Please check [this](https://github.com/hiqdev/asset-packagist/issues/97), so when they fix the search, update README where
```
composer require "npm-asset/fortawesome--fontawesome-free:~5.7.1"
```
to
```
composer require "npm-asset/@fortawesome/fontawesome-free:~5.7.1"
```

**Comment by rmrevin** [comment]
Thanks.

**Pull request merged**